### PR TITLE
Add corsOrigins to config

### LINKED ISSUE #879: Research (and implement if possible) enabling CORS on the HTTP gateways
Documentation for https://github.com/api3dao/airnode/issues/1200

### PR DESCRIPTION
**Pull request by vponline**
Closes #879

### CODE REVIEW

File: docs/airnode/v0.8/reference/deployment-files/config-json.md [hunk 1]
```diff
@@ -442,6 +444,11 @@ must have a length of between 30 - 120 characters. Do not use the same key for
 number of serverless functions serving HTTP gateway requests running at the same
 time. When omitted, there is no maximum concurrency set.
 
+#### `httpGateway.corsOrigins`
+
+(only if enabled) - A list of allowed origins. An empty array (`[]`) can be used
+to disable CORS and the wildcard (`['*']`) can be used to allow all origins.
+
```
Comment (wkande) [inline docs/airnode/v0.8/reference/deployment-files/config-json.md:459]: @vponline - Question here. So if the field `httpGateway.corsOrigins` is not present at all (in config.json) is CORS disabled as well?

**Review by wkande — APPROVED** [review]
Made a small change based on our Slack chat and following concerns of GitHub issue https://github.com/api3dao/api3-docs/issues/960

File: docs/airnode/v0.8/reference/deployment-files/config-json.md (hunk 1 above)
Comment (vponline) [reply docs/airnode/v0.8/reference/deployment-files/config-json.md:459]: Yes it is disabled, but the config validation will fail if `httpGateway` or `httpSignedDataGateway` is enabled but missing the `corsOrigins`. 

So the change you made is accurate, thanks!

Comment (wkande) [reply docs/airnode/v0.8/reference/deployment-files/config-json.md:459]: Will merge now

**Pull request merged**